packages/core/src/modules/connections/handlers/DidExchangeResponseHandler.ts

```diff
@@ -95,10 +97,14 @@ export class DidExchangeResponseHandler implements Handler {
     // TODO: should we only send complete message in case of autoAcceptConnection or always?
     // In AATH we have a separate step to send the complete. So for now we'll only do it
     // if auto accept is enable
```

Comment:
Not in your pr, but: 

```suggestion
    // if auto accept is enabled
```